recipes/sqt/meta.yaml

```diff
@@ -0,0 +1,31 @@
+about:
+  home: https://bitbucket.org/marcelm/sqt
+  license: MIT
+  summary: 'Command-line tools for the analysis of high-throughput sequencing data'
+package:
+  name: sqt
+  version: "0.6.0"
+source:
+  fn: sqt-0.6.0.tar.gz
+  url: https://pypi.python.org/packages/source/s/sqt/sqt-0.6.0.tar.gz
+  md5: b1b9f6a1aedb3c462d87dc8de284497f
+requirements:
+  run:
+    - python >=3.3
```

Comment:
With this, builds for Python 2 will fail (which is of course correct, but our build system can't distinguish this). The pattern to solve this is to add a skip instruction for Python 2 to the build section (see [here](https://github.com/bioconda/bioconda-recipes/blob/5a178309d890030508b64ed80cb1e778b4862e44/recipes/manta/meta.yaml)).